extensions/variants.go

```diff
@@ -218,6 +222,20 @@ func parseFuncName(compoundName string) (name string, args ArgumentList) {
 	return name, args
 }
 
+func minArgumentCount(paramTypeList ArgumentList, variadicBehavior *VariadicBehavior) int {
+	if variadicBehavior == nil {
+		return len(paramTypeList)
+	}
+	return variadicBehavior.Min
```

Comment:
shouldn't this be `len(paramTypeList) + variadicBehavior.Min` ?

Reply:
I think it should actually be len(paramTypeList) + variadicBehavior.Min - 1, no?

Reply:
going by the example of the [`or` function](https://github.com/substrait-io/substrait/blob/main/extensions/functions_boolean.yaml)

it has one parameter and a `Min` of 0 for the variadic. As a result, the Minimum number of arguments is 1, `len(paramTypeList) + variadicBehavior.Min`

If you use the `- 1` then it would have a minimum of 0 which seems wrong, right?

Reply:
I think this should be "len(paramTypeList) + variadicBehavior.Min".
Fixed it